### PR TITLE
Update activate-using-active-directory-based-activation-client.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10213

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 1]
```diff
@@ -20,18 +20,11 @@ ms.collection: highpri
...
->- Windows Server 2016
->- Windows Server 2019
->- Office 2013*
->- Office 2016*
->- Office 2019*
+**Applies to**
+-   Windows 8.1 or later
+-   Windows Server 2012 or later
+-   Microsoft Office 2013 or later
+
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:27]:
```suggestion
```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 2]
```diff
@@ -109,14 +102,16 @@ When a reactivation event occurs, the client queries AD DS for the activation o
     **Figure 15**. Choosing how to activate your product
 
     > [!NOTE]
-    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version-specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.
+    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version-specific Office Volume License Pack on the server where the Volume Activation Server Role is installed. For more details see [Activate volume licensed versions of Office by using Active Directory](/deployoffice/vlactivation/activate-office-by-using-active-directory)
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:105]:
```suggestion
    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version-specific Office Volume License Pack on the server where the Volume Activation Server Role is installed. For more details, see [Activate volume licensed versions of Office by using Active Directory](/deployoffice/vlactivation/activate-office-by-using-active-directory).

```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 3]
```diff
@@ -109,14 +102,16 @@ When a reactivation event occurs, the client queries AD DS for the activation o
...
     > 
     > 
     > - [Office 2013 VL pack](https://www.microsoft.com/download/details.aspx?id=35584)
     > 
     > - [Office 2016 VL pack](https://www.microsoft.com/download/details.aspx?id=49164)
     >
     > - [Office 2019 VL pack](https://www.microsoft.com/download/details.aspx?id=57342)
+    >
+    > - [Office LTSC 2021 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=103446)
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:114]:
```suggestion
    > - [Office LTSC 2021 VL pack](https://www.microsoft.com/download/details.aspx?id=103446)

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**